File: problem_solving_for_beginners/array_problems/python/reverse_array.py
#!/usr/bin/python3
"""Reverse array elements"""
arr = []
n = int(input("Enter a size of array: "))

for i in range(n):
    i = int(input("Enter element of array: "))
    arr.append(i)

#write your code here
half =  n // 2
lastindex = n - 1

for i in range(half):
    tmp = arr[i]
    arr[i] = arr[lastindex - i]
    arr[lastindex - i] = tmp

for i in arr:
    print(i)

File: problem_solving_for_beginners/array_problems/c-programming/reverse_array.c
#include<stdio.h>
/**
 * main - Reverse elements of array
 *
 * Return: 0 Always (success)
 */
int main(void)
{
	int arr[10], n, i;
	printf("Enter the size of array: ");
    scanf("%d", &n);

    for(i = 0; i < n; i++)
    {
        printf("Enter the size of array: ");
        scanf("%d", &arr[i]);
    }

    //Write your code here
    int tmp, lastindex, half;

    lastindex = n - 1;
    half = n / 2;
    for (i = 0; i < half; i++)
    {
        tmp = arr[i];
        arr[i] = arr[lastindex - i];
        arr[lastindex - i] = tmp;
    }

    for (i = 0; i < n; i++)
        printf("%d", arr[i]);

    printf("\n");

    return 0;
}